tests/ssg_test_suite/oscap.py

```diff
@@ -424,6 +431,9 @@ def final(self):
             logging.info("Rebooting domain '{0}' before final scan."
                          .format(self.environment.domain_name))
             self.environment.reboot()
+            logging.info("Waiting for {0} seconds to let the system finish startup and delete possible Dracut temporary files"
+                         .format(self.time_to_finish_startup))
```

Comment:
Now I think the printed message is too specific, :-P.

The comment above definition of `time_to_finish_startup` should be enough to justify the delay.
For the log message itself, I think just `Waiting for {0} seconds to let the system finish startup` is enough.